src/app/not-found.tsx

```
@@ -0,0 +1,84 @@
+              fill="#3B82F6"
+              d="M39.5,-65.3C50.2,-55.3,57.1,-42.4,62.6,-28.8C68.1,-15.2,72.3,-0.9,71.8,13.5C71.2,27.8,66,42.3,56.3,53.2C46.6,64.1,32.4,71.4,17.8,73.5C3.2,75.5,-11.8,72.3,-24.8,66.3C-37.8,60.3,-48.8,51.5,-57.3,40.5C-65.8,29.5,-71.8,16.3,-73.7,2.2C-75.5,-12,-73.2,-27,-65.3,-38C-57.4,-49,-43.9,-56,-30.8,-64.8C-17.7,-73.5,-4.9,-84,6.8,-94.3C18.5,-104.6,37,-105.7,39.5,-65.3Z"
+              transform="translate(100 100)"
+            />
+          </svg>
+          <svg
+            className="absolute left-1/2 top-1/2 size-32 -translate-x-1/2 -translate-y-1/2"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="white"
+            strokeWidth={2}
+          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)